DSCResources/MSFT_NetworkTeam/en-US/MSFT_NetworkTeam.psd1

```diff
@@ -0,0 +1,22 @@
+ConvertFrom-StringData @'    
```

Comment:
I'm not sure this file is actually needed if we keep the en-us culture in the base file. Best to avoid having to change the string in two different places?